Change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    samples2 = fn(samples)
</pre><p>Here two types of samples are used, so we can give meaningful names, something like</p><p>samples -&gt; uniform_samples</p><p>samples2 -&gt; transformed_samples (or any meaning full name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>